src/providers/auth-provider.tsx

```diff
@@ -258,19 +252,33 @@ export const AuthProvider = ({ children }: { children: ReactNode }) => {
         subOrgId = subOrg.subOrganizationId
       }
 
-      const oauthResponse = await oauth({
-        credential,
-        targetPublicKey: `${authIframeClient?.iframePublicKey}`,
-        targetSubOrgId: subOrgId,
-      })
-      const credentialResponse = await authIframeClient?.injectCredentialBundle(
-        oauthResponse.credentialBundle
-      )
+      if (authIframeClient?.iframePublicKey) {
```

Comment:
looks like this is something we can template + abstract away via library as well :)

Reply:
exactly!